tests/src/test/scala/cats/tests/VarianceSuite.scala

```diff
@@ -23,24 +25,45 @@ class VarianceSuite extends CatsSuite {
   }
 
   test("Auto-variance should widen a bifunctor automatically") {
-    def shouldInfer[F[_, _]: Bifunctor](fi: F[Int, Int]) =
+    def shouldInfer[F[_, _]: Bifunctor](fi: F[Int, Int]): F[Either[Bar, Baz.type], Either[Bar, Baz.type]] =
```

Comment:
think that the test would be more readable if explicitly typed

Reply:
The idea was to assert here that scalac can infer the correct result type :)

Reply:
The test is still checking that the conversion happens at calls site
```scala
def inferred[F[_, _]: Bifunctor](fi: F[Int, Int]): F[Either[Foo, Foo], Either[Foo, Foo]] = shouldInfer[F](fi)
```
If we remove the `import cats.conversions.all._` the compilation fails also in this version, so the test is still valid. I don't have strong preferences about the type annotation.

Reply:
Now if I think some more, between the two not having types is still more useful, thanks @LukaJCB